src/MockeryTools/Http/HttpRequestOptionsMatcher.php

```diff
@@ -17,12 +17,12 @@ class HttpRequestOptionsMatcher extends MatcherAbstract
     /**
      * @var array<string, string>
      */
-    private $expectedHeaders;
+    private array $expectedHeaders;
 
     /**
      * @var mixed[]|null
      */
-    private $expectedRequestData;
+    private ?array $expectedRequestData = null;
```

Comment:
Reduntant intialization, it's already initialized in the constructor
```suggestion
    private ?array $expectedRequestData;
```